TestFramework/ExtensionMethods.cs

```diff
@@ -77,6 +78,44 @@ public static string GetElementName(this ScenarioContext scenarioContext)
             return value;
         }
 
+        public static bool IsDisplayed(this IWebDriver webdriver, By findBy, ScenarioContext scenarioContext, TimeSpan? waitPeriod = null)
```

Comment:
@vsdevelope, how is this different from IsElementVisible()? which is in the same file? Which also returns el.Displayed..

Reply:
IsElementVisible checks for 2 properties of IWebElement
1. IWebElement.Displayed and IWebElement.Enabled

Where as IsDisplayed only checks for IWebElement.Displayed property. It returns true even when the element is disabled.